prime-router/src/main/kotlin/cli/main.kt

```diff
@@ -284,4 +250,99 @@ class RouterCli : CliktCommand(
     }
 }
 
-fun main(args: Array<String>) = RouterCli().main(args)
\ No newline at end of file
+fun listSchemas(metadata: Metadata) {
+    println("Current Hub Schema Library")
+    var formatTemplate = "%-25s\t%-10s\t%s"
+    println(formatTemplate.format("Schema Name", "Topic", "Description"))
+    metadata.schemas.forEach {
+        println(formatTemplate.format(it.name, it.topic, it.description))
+    }
+}
+
+class ListSchemas : CliktCommand(
+    name = "list",
+    help = "list known schemas, clients, and services"
+) {
+    fun listOrganizations(metadata: Metadata) {
+        println("Current Clients (Senders to the Hub)")
+        var formatTemplate = "%-18s\t%-10s\t%s"
+        println(formatTemplate.format("Organization Name", "Client Name", "Schema Sent to Hub"))
+        metadata.organizationClients.forEach {
+            println(formatTemplate.format(it.organization.name, it.name, it.schema))
+        }
+        println()
+        println("Current Services (Receivers from the Hub)")
+        formatTemplate = "%-18s\t%-10s\t%-25s\t%s"
+        println(formatTemplate.format("Organization Name", "Service Name", "Schema Sent by Hub", "Filters Applied"))
+        metadata.organizationServices.forEach {
+            println(
+                formatTemplate.format(
+                    it.organization.name, it.name, it.schema,
+                    it.jurisdictionalFilter.joinToString()
+                )
+            )
+        }
+    }
+
+    override fun run() {
+        val metadata = Metadata(Metadata.defaultMetadataDirectory)
+        println()
+        listSchemas(metadata)
+        println()
+        listOrganizations(metadata)
+        println()
+    }
+}
+
+class GenerateDocs : CliktCommand(
+    help = """
+    generate documentation for schemas
+
+    By default, this will generate documentation for all known schemas. To
+    generate docs for a particular schema, use the --input-schema option.
+    """
+) {
+    private val inputSchema by option("--input-schema", metavar = "<schema_name>", help = "schema to document")
+    private val includeTimestamps by
+    option("--include-timestamps", help = "include creation time in file names")
+        .flag(default = false)
+    private val outputFileName by option("--output", metavar = "<path>", help = "write documentation to this file (should not include extension)")
+    private val defaultOutputDir = "docs/schema_documentation"
+    private val outputDir by option("--output-dir", metavar = "<path>", help = "interpret `--output` relative to this directory (default: \"$defaultOutputDir\")")
+        .default(defaultOutputDir)
```

Comment:
I could not for the life of me figure out how to make `helpTags` and `defaultForHelp` actually do their job and print the default value in the help text, so I did this. If anyone knows how to make Clikt do it, please fill me in!